### DISCUSSION TITLE
Password Sharing Features?

### DISCUSSION THREAD

**Discussion by Theknight2015** [dani-garcia/vaultwarden#3531] [Q&A] [answered]
I'm a total noob at Vaultwarden and Bitwarden but one thing I'm not seeing that I have been looking for is a way to share a single username and password with a single user. I see that you can move passwords between collections (Where multiple people have access) and organizations (Where multiple collections/people have access) but I'm not seeing a way to share something with a single person without making a collection for every single person you want to share things with.

I'm coming from Lastpass where there is an option to "Share" an item and then you select the user or enter the email from the drop bar or search field and it shares that item with only that person. Then in your vault you can see what items are shared and with whom they're shared to.

I'm assuming this is a feature of Vaultwarden/Bitwarden in some regard but I'm not sure how to do it from the web view, web plugin or the desktop app as I can't find the option to share with anything other than a collection or organization.

Any help would be greatly appreciated.

**Comment by stefan0xC** [comment, marked as answer]
[Vault Item Sharing](https://community.bitwarden.com/t/vault-item-sharing/238) is a long-requested feature that is part of [Bitwarden's roadmap for 2023](https://community.bitwarden.com/t/bitwarden-roadmap/12865). So it's not implemented yet but hopefully it will be soon.

**Comment by BlackDex** [comment]
There isn't anything in comparison as far as i know.
One way would be to use Send. But that's not sharing that specific cipher to an other user.

This would need to be something built into the Bitwarden clients, since they need to support this kind of feature.

**Comment by Theknight2015** [comment]
@stefan0xC and @BlackDex thank you for the information. I will just have to wait for a newer version I suppose. Creating individual collections for everyone or sending/transferring credentials is not ideal for my situation as I have a large family. I appreciate both of you for responding.

**Comment by kenlyle2** [comment]
Following.  Looking for cost-effective password sharing.  Zoho Vault's Standard Plan is winning for me atm.

**Comment by schmurtzm** [comment]
Following, we can't adopt this solution while this feature does not exist.

**Comment by kpfleming** [comment]
There's nothing to 'follow' here; this question has been answered, this functionality does not exist in the Bitwarden clients yet.

**Comment by MembershipExcel** [comment]
over 18 month later, is there an update on this feature ? 
I know it exist for the familly Edition (with licence fee) but maybe it available within vaulwarden ?

**Reply by schmurtzm** [reply]
It's not in the 2024 roadmap anymore so I don't have big hope...
@stefan0xC , any news on this ?
I've read some posts [here](https://community.bitwarden.com/t/vault-item-sharing/238/130) but the last "official" post was "it's in the roadmap" 🤷‍♂️

**Reply by BlackDex** [reply]
That is where organizations and collections are for? Or are you talking about something totally different?

If this is about individual password/item sharing, then first the key per cipher needs to work correctly for all clients. And i think Bitwarden had some issues with that.

**Reply by schmurtzm** [reply]
Not the collections indeed, the objective is to share an item without creating a collection : https://community.bitwarden.com/t/vault-item-sharing/238
I hope it will come one day but I don't even see that in the roadmap. Honestly it's a mess to create a collection with each person you need to share a single password....

**Reply by BlackDex** [reply]
First Bitwarden needs to add this feature before Vaultwarden can implement this.

**Comment by JavaZeroo** [comment]
any update?

**Reply by BlackDex** [reply]
My answer will be the same as in 2024. First Bitwarden needs to implement this before Vaultwarden can. And as far as i know this feature doesn't exist.

**Comment by logiczny** [comment]
So... we're still waiting for Bitwarden, huh?

**Comment by g0lg0-93** [comment]
https://bitwarden.com/help/sharing/ << doesn't this feature now exist?

**Reply by dubwoc** [reply]
No, that linked feature is about "sharing" passwords or moving passwords into Organizations. The original bug is about being able to share a link with someone who does not have a login to your vaultwarden so they can see the item you shared. Think of it like a temporary unauthenticated but obfuscated shared password. 

I feel like bitwarden's use of the word share is a bit odd in that documentation.

**Comment by ICantThinkOfAName-tech** [comment]
doesn't the feature exist now? https://bitwarden.com/products/send/

**Reply by stefan0xC** [reply]
While you can use this feature to copy/paste text and thus share a password that way, the idea behind the [requested password sharing feature](https://community.bitwarden.com/t/vault-item-sharing-different-from-the-current-organizations-collections-implementation/238) was to make it independent from organizations/collections to share individual items to others via a Share button.

edit: as far as I've looked [at their current roadmap](https://community.bitwarden.com/c/feature-requests/roadmap/l/latest?board=default) it seems like [a feature like that is currently in progress](https://community.bitwarden.com/t/share-an-existing-vault-item-with-a-link/100890).
